articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,23 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
+- IPv6 clusters
+- [Service Principals](/azure/devops/integrate/get-started/authentication/service-principal-managed-identity)
+  - *Note: must be MSI-based, either system managed identity or user managed identity*
+- Disk encryption sets
+- CustomCATrustCertificates
+- [Start Stop mode](./start-stop-cluster.md)
+- [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) (deprecated)
```

Comment:
Why is this listed since it's deprecated and we've clearly referred customers to Workload Identity?

Reply:
ack, makes sense to remove